spec/jobs/send_webhook_job_spec.rb

```diff
@@ -122,6 +124,29 @@
     end
   end
 
+  context 'when webhook_type is payment_provider_customer_checkout_url' do
+    let(:webhook_service) { instance_double(Webhooks::PaymentProviders::CustomerCheckoutService) }
+    let(:customer) { create(:customer) }
+
+    before do
+      allow(Webhooks::PaymentProviders::CustomerCheckoutService).to receive(:new)
+        .with(customer, checkout_url: 'https://example.com')
+        .and_return(webhook_service)
+      allow(webhook_service).to receive(:call)
+    end
+
+    it 'calls the webhook service' do
+      subject.perform_now(
```

Comment:
RSpec/NamedSubject: Name your test subject if you need to reference it explicitly. (https://rspec.rubystyle.guide/#use-subject, https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/NamedSubject)